src/main/java/com/eleks/academy/whoami/core/state/ProcessingQuestion.java

```diff
@@ -39,13 +43,39 @@ public void ask(String player, PlayerAction.Action question) {
 		throw new GameException("Not implemented");
 	}
 
-	;
+	public void answer(String player, String value) {
+		if (turn.getCurrentTurn().get(0).getValue() != null) {
```

Comment:
you can add into turn methods such as `isQuestionPresent` because who is asking question `turn.getCurrentTurn().get(0).getValue() != null` only turn should know